lib/fastlane/plugin/sentry/actions/sentry_set_commits.rb

```diff
@@ -63,6 +64,10 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :commit,
                                        description: "Commit spec, see `sentry-cli releases help set-commits` for more information",
                                        optional: true)
+          FastlaneCore::ConfigItem.new(key: :ignore_missing,
+                                       description: "When enabled, if the previous release commit was not found in the repository, will create a release with the default commits count",
```

Comment:
```suggestion
                                       description: "When enabled, if the previous release commit was not found in the repository, will create a release with the default commits count (or the one specified with `--initial-depth`) instead of failing the command.",
```